core/src/main/java/bisq/core/locale/BankUtil.java

```diff
@@ -157,9 +157,9 @@ public static String getAccountNrLabel(String countryCode) {
             case "HK":
                 return Res.get("payment.accountNr");
             case "NO":
-                return "Kontonummer"; // do not translate as it is used in Norwegian only
+                return "Kontonummer"; // do not translate as it is used in norwegian and swedish only
```

Comment:
As it looks like the label is exactly the same I think we could merge those two cases.